skypy/halo/tests/test_quenching.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+from scipy import stats
+from collections import Counter
+
+from skypy.halo.quenching import environment_quenched, mass_quenched
+
+
+def test_environment_quenched():
+    # Test the quenching process follows a binomial distribution
+    n, p = 1000, 0.7
+    quenched = environment_quenched(n, p)
+    number_quenched = Counter(quenched)[0]
```

Comment:
```suggestion
    number_quenched = Counter(quenched)[True]
```
`Counter` is a dictionary subclass so this is the correct way to return the count for `True`

Reply:
Because `Counter` is a dictionary there is no guarantee that the `True` count is the first entry i.e. `Counter(quenched)[0]` could sometimes return the `False` count. I think this is why your test was failing and actually had nothing to do with switching to `alternative='two-sided'`

Reply:
Awesome, thanks